docker-compose.yml

```diff
@@ -1,19 +1,25 @@
-version: '2'
+version: '3'
 
 services:
   redis:
     image: redis
   db:
-    image: postgres:10
+    image: postgres
     environment:
       POSTGRES_PASSWORD: password
-    ports:
-      - "5432:5432"
```

Comment:
Why this removal?

Reply:
to make it closer to https://docs.docker.com/compose/django/